experimenter/experimenter/nimbus-ui/src/components/PageSummary/TableSignoff/index.tsx

```diff
@@ -1,26 +1,61 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import React from "react";
+import React, { ChangeEvent } from "react";
 import { Card, Table } from "react-bootstrap";
+import Form from "react-bootstrap/Form";
 import LinkExternal from "src/components/LinkExternal";
 import { EXTERNAL_URLS } from "src/lib/constants";
 import { getExperiment_experimentBySlug_signoffRecommendations } from "src/types/getExperiment";
 
 type TableSignoffProps = {
   signoffRecommendations: getExperiment_experimentBySlug_signoffRecommendations | null;
+  legalSignoff: boolean | null;
+  qaSignoff: boolean | null;
+  vpSignoff: boolean | null;
+  onLegalSignoffChange: (value: boolean) => void;
+  onQaSignoffChange: (value: boolean) => void;
+  onVpSignoffChange: (value: boolean) => void;
 };
 
-const TableSignoff = ({ signoffRecommendations }: TableSignoffProps) => (
+const handleLegalSignoffChange =
+  (onLegalSignoffChange: (value: boolean) => void) =>
+  (event: ChangeEvent<HTMLInputElement>) => {
+    onLegalSignoffChange(event.target.checked);
+  };
+
+const handleQaSignoffChange =
+  (onQaSignoffChange: (value: boolean) => void) =>
+  (event: ChangeEvent<HTMLInputElement>) => {
+    onQaSignoffChange(event.target.checked);
+  };
+
+const handleVpSignoffChange =
+  (onVpSignoffChange: (value: boolean) => void) =>
+  (event: ChangeEvent<HTMLInputElement>) => {
+    onVpSignoffChange(event.target.checked);
+  };
+
+const TableSignoff = ({
+  signoffRecommendations,
+  legalSignoff,
+  qaSignoff,
+  vpSignoff,
+  onLegalSignoffChange,
+  onQaSignoffChange,
+  onVpSignoffChange,
+}: TableSignoffProps) => (
   <Card.Body>
     <Table data-testid="table-signoff">
       <tbody>
         <tr data-testid="table-signoff-qa">
           <th className="border-top-0">QA Sign-off</th>
-          <td className="border-top-0">
+          <td className="border-top-0 d-flex">
+            <Form.Check
+              type="checkbox"
+              data-testid="is-qasignoff-checkbox"
+              checked={!!qaSignoff}
+              onChange={handleQaSignoffChange(onQaSignoffChange)}
+            />
             {signoffRecommendations?.qaSignoff && (
-              <span className="text-success">Recommended: </span>
+              <span className="text-success">Recommended:&nbsp; </span>
```

Comment:
```suggestion
              <span className="text-success">Recommended: {" "}</span>
```

if this doesnt work, `<><span>..></span>{" "}</>`

Reply:
nope both are not working